backend/src/main/java/com/example/backend/service/WorkService.java

```diff
@@ -15,4 +17,22 @@ public class WorkService {
     public List<WorkDto> getAllWorksByCarId (String carId) {
         return workRepository.findAllByCarId(carId);
     }
+
+    public WorkDto createWork(WorkDto workDto) {
+        Work work = new Work(workDto.id(), workDto.carId(), workDto.type(), workDto.mileage(), workDto.date(), workDto.price());
+        workRepository.save(work);
+        return workDto;
```

Comment:
The input parameter is returned without any changes.

Reply:
I don’t see the point in returning something else, since, for example, the id is already set at the controller level, that is, we already have a full-fledged object that will be written into the database. And we return it as a result.